lxc/completion.go

```diff
@@ -377,12 +375,9 @@ func (g *cmdGlobal) cmpInstanceSetKeys(instanceName string) ([]string, cobra.She
 	for configKey := range instance.Config {
 		// We only want to return the intersection between allInstanceConfigKeys and configKeys to avoid returning the full instance config.
 		_, exists := keySet[configKey]
-		if exists {
-			if shared.StringHasPrefix(configKey, instancetype.ConfigKeyPrefixesAny...) {
-				configKeys = append(configKeys, configKey)
-			} else if instanceType == string(api.InstanceTypeContainer) && shared.StringHasPrefix(configKey, instancetype.ConfigKeyPrefixesContainer...) {
-				configKeys = append(configKeys, configKey)
-			}
+		knownPrefixes := append(instancetype.ConfigKeyPrefixesAny, instancetype.ConfigKeyPrefixesContainer...)
```

Comment:
why are we building this slice for every iteration of the loop? It seems static?